### PR TITLE
Show warning in suggestion dialog when GEN_DATA INPUT_FORMAT missing or unsupported.

### PR DESCRIPTION
**Pull request by DanSava**
**Issue**

Resolves #4965

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Updated documentation
- [x] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: tests/unit_tests/c_wrappers/res/enkf/test_ensemble_config.py [hunk 1]
```diff
@@ -144,7 +136,41 @@ def test_gen_data_node(gen_data_str, expected):
...
+            id="Missing INPUT_FORMAT no format",
+        ),
+        pytest.param(
+            "",
+            "Valid",
+            id="Missing INPUT_FORMAT no key and format",
+        ),
+    ],
+)
+def test_gen_data_node_input_format(input_format, expected):
```
Comment (eivindjahren) [inline tests/unit_tests/c_wrappers/res/enkf/test_ensemble_config.py:159]: I believe expected is unused here.

**Review by eivindjahren — CHANGES_REQUESTED** [review]
Just a minor comment on the test, otherwise we are ready to go :)

File: tests/unit_tests/c_wrappers/res/enkf/test_ensemble_config.py (hunk 1 above)
Comment (DanSava) [reply tests/unit_tests/c_wrappers/res/enkf/test_ensemble_config.py:159]: That seems to be the case.

**Review by eivindjahren — APPROVED** [review]
:100:

**Comment by DanSava** [comment]
Jenkins test this please!

**Pull request merged**